app/cbt/utils/text_manipulations.py

```diff
@@ -0,0 +1,117 @@
+from config.get_config import config_object
+import aiohttp
+from utils.utils import logger
+
+
+async def get_languages() -> dict:
+    async with aiohttp.ClientSession(
+        base_url=config_object.libretranslate_server_config.url
+    ) as session:
+        languages: list
+        async with session.get("/languages") as response:
+            match response.status:
+                case 200:
+                    logger.debug(f"Got response from {response.real_url}")
+                    try:
+                        languages = await response.json()
+                    except Exception as e:
+                        logger.error(f"Failed to fetch languages with error {e}")
+                        raise e
+                case _:
```

Comment:
Imagine using a pattern match here instead of just 

``` python
if response.ok:
```

Did I do that in my code? No. Do as I say, not as I do mr "tech led"

Reply:
cry